content/en/logs/explorer/insights.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Logs from Watchdog Insights
+kind: documentation
+description: 'Rely on Watchdog Insights to surface meaningful signals within your current seach context.'
+aliases:
+    - /logs/explorer/insights
+further_reading:
+    - link: 'logs/explorer/group'
+      tag: 'Documentation'
+      text: 'Group queried logs'
+    - link: 'logs/explorer/visualize'
+      tag: 'Documentation'
+      text: 'Create visualizations from logs'
+    - link: '/logs/explorer/export'
+      tag: 'Documentation'
+      text: 'Export Log Explorer views'
+---
+
+## Overview
+
+Insights are a selection of meaningful signals that standout within your current search context (time and tags).
+
+In the Log Explorer they allow our machine learning engine, Watchdog, to surface logs (caracterized by their patterns or specific tags) that have been flagged by our data science capabilities (like Log Anomaly Detection or Error Outliers Detection).
+
+{{< img src="logs/explorer/insights/insights-carousel.png" alt="Watchdog Insights in Log Explorer" style="width:100%;" >}}
+
+### Log Anomaly Detection
+
+Ingested logs are analyzed at the intake level where Watchdog performs aggregations on detected patterns as well as `environment`, `service`,`source` and `status`tags.
```

Comment:
```suggestion
Ingested logs are analyzed at the intake level where Watchdog Insights perform aggregations on detected patterns as well as `environment`, `service`,`source` and `status` tags.
```

Reply:
Watchdog Insights is the vehicule for surfacing interesting things in context (Log Explorer, traces Explorer etc.).

Here it is about computations and it's the work of the Machine Learning engine, Watchdog itself.

To make this distinction clearer, we are working on reframing these so as to have: Watchdog (for the engine), Insights powered by Watchdog, Alerts powered by Watchdog etc.